src/Discord/Parts/Channel/Message.php

```diff
@@ -667,14 +668,15 @@ public function crosspost(): ExtendedPromiseInterface
      *
      * @param string|MessageBuilder $message Reply message to send after delay.
      * @param int                   $delay   Delay after text will be sent in milliseconds.
+     * @param TimerInterface        &$timer  Delay timer passed by reference.
      *
      * @return ExtendedPromiseInterface<Message>
      */
-    public function delayedReply($message, int $delay): ExtendedPromiseInterface
+    public function delayedReply($message, int $delay, TimerInterface &$timer = null): ExtendedPromiseInterface
```

Comment:
There is no need for requiring a TimerInterface to be passed as $timer, since it's value will be replaced by a TimerInterface value anyways, overwriting whatever value passed.